tests/commands/test_changelog_command.py

```diff
@@ -514,3 +516,311 @@ def test_changelog_incremental_with_release_candidate_version(
         out = f.read()
 
     file_regression.check(out, extension=".md")
+
+
+@pytest.mark.usefixtures("tmp_commitizen_project")
+def test_changelog_with_filename_as_empty_string(mocker, changelog_path, config_path):
+
+    with open(config_path, "a") as f:
+        f.write("changelog_file = true\n")
+
+    create_file_and_commit("feat: add new output")
+
+    testargs = ["cz", "changelog"]
+    mocker.patch.object(sys, "argv", testargs)
+    with pytest.raises(NotAllowed):
+        cli.main()
+
+
+@pytest.mark.usefixtures("tmp_commitizen_project")
+@pytest.mark.freeze_time("2022-02-13")
+@mock.patch("commitizen.git.GitTag.date", "2022-02-13")
```

Comment:
Should we use `mocker` fixture in the test function? Or, would it cause any problem?